src/poliastro/plotting.py

```diff
@@ -160,12 +160,14 @@ def _project(self, rr):
     def _redraw_attractor(self, min_radius=0 * u.km):
         radius = max(self._attractor.R.to(u.km).value, min_radius.to(u.km).value)
         color = BODY_COLORS.get(self._attractor.name, "#999999")
-        self._attractor_radius = radius
 
         for attractor in self.ax.findobj(match=mpl.patches.Circle):
             attractor.remove()
 
-        self.ax.add_patch(mpl.patches.Circle((0, 0), radius, lw=0, color=color))
+        if radius < self._attractor_radius:
```

Comment:
Shouldn't this have a unit, as it's done in the other two cases (`np.inf * u.km`)? I'm just asking, if the code works like this then it's fine for me.

Reply:
Although it was working, but then it seems more uniform. Thanks for the suggestion.